parlai/core/torch_ranker_agent.py

```diff
@@ -26,40 +27,42 @@ def add_cmdline_args(cls, argparser):
             help='The source of candidates during training '
                  '(see TorchRankerAgent._build_candidates() for details).')
         agent.add_argument(
-            '-ecands', '--eval-candidates', type=str,
+            '-ecands', '--eval-candidates', type=str, default='inline',
             choices=['batch', 'inline', 'fixed', 'vocab'],
-            help='The source of candidates during evaluation (defaults to the same'
-                 'value as --candidates if no flag is given)')
+            help='The source of candidates during evaluation (defaults
+                 'inline candidates).')
         agent.add_argument(
             '-fcp', '--fixed-candidates-path', type=str,
             help='A text file of fixed candidates to use for all examples, one '
                  'candidate per line')
         agent.add_argument(
             '--fixed-candidate-vecs', type=str, default='reuse',
-            help="One of 'reuse', 'replace', or a path to a file with vectors "
-                 "corresponding to the candidates at --fixed-candidates-path. "
-                 "The default path is a /path/to/model-file.<cands_name>, where "
-                 "<cands_name> is the name of the file (not the full path) passed by "
-                 "the flag --fixed-candidates-path. By default, this file is created "
-                 "once and reused. To replace it, use the 'replace' option.")
+            help='One of "reuse", "replace", or a path to a file with vectors '
+                 'corresponding to the candidates at --fixed-candidates-path. '
+                 'The default path is a /path/to/model-file.<cands_name>, where '
+                 '<cands_name> is the name of the file (not the full path) passed by '
+                 'the flag --fixed-candidates-path. By default, this file is created '
+                 'once and reused. To replace it, use the "replace" option.')
+        agent.add_argument(
+            '--encode-candidate-vecs', type='bool', default=False,
+            help='Cache and save the encoding of the candidate vecs. This '
+                 'might be used when interacting with the model in real time '
+                 'or evaluating on fixed candidate set when the encoding of '
+                 'the candidates is independent of the input.')
         agent.add_argument(
             '--train-predict', type='bool', default=False,
             help='Get predictions and calculate mean rank during the train '
-                 'step. Turning this on may slow down training.'
-        )
+                 'step. Turning this on may slow down training.')
         agent.add_argument(
             '--ignore-bad-candidates', type='bool', default=False,
             help='Ignore examples for which the label is not present in the '
-                 'label candidates. Default behavior results in RuntimeError. '
-        )
+                 'label candidates. Default behavior results in RuntimeError. ')
```

Comment:
lol are you and i having a parenthesis war?

Reply:
haha i was just making it match the other args but i can make it do whatever

Reply:
consistency with the file is better. your change is good